docs/audio.rst

```diff
@@ -104,11 +315,11 @@ samples. When reading reaches the start or the mid-point of the buffer, it
 triggers a callback to fetch the next ``AudioFrame`` which is then copied into
 the buffer. This means that a sound source has under 4ms to compute the next
 ``AudioFrame``, and for reliable operation needs to take less 2ms (which is
-32000 cycles, so should be plenty).
+32k cycles in micro:bit V1 or 128k in V2, so should be plenty).
```

Comment:
32k could mean 32768.  I'd leave it as 32000.

Reply:
Yeah, good call, it'll change I'll use the full figure. 👍